### PR TITLE
Cl reduce binaries

### PR DESCRIPTION
**Pull request by gpx1000**
Reducing the binaries as part of phase 2.

### CODE REVIEW

**Comment by lakshmih** [comment]
Need time to review

**Comment by lakshmih** [comment]
Waiting for merge conflicts to be resolved before reviewing.

**Review by bashbaug — COMMENTED** [review]
There are more diffs here than I was expecting to see because it looks like some files were deleted and re-added vs. moved.  If the files are moved instead are there fewer diffs?

File: test_conformance/api/main.cpp [hunk 1]
```diff
@@ -150,9 +150,59 @@ test_definition test_list[] = {
     ADD_TEST(negative_get_platform_ids),
 };
 
+test_definition test_list_with_check[] = {
+    ADD_TEST(timer_resolution_queries),
+    ADD_TEST(device_and_host_timers),
+};
```
[the remark this answers is not in the document]
Comment (gpx1000) [reply test_conformance/api/main.cpp:156]: Sure, that sounds like a good idea.

**Comment by gpx1000** [comment]
> There are more diffs here than I was expecting to see because it looks like some files were deleted and re-added vs. moved. If the files are moved instead are there fewer diffs?

Sadly, each were originally moved files.  During the merge conflict changes github seems to have labeled them as delete/creations instead of moves.

**Comment by lakshmih** [comment]
After careful review, we believe that reducing the number of binaries does not necessarily reduce complexity or runtime of the CTS suite.  On the contrary, combining binaries would potentially make it harder to analyze for failures and also introduce overhead in terms of the need for updating test infrastructure and scripts.  Further, the change is combining features that don't necessarily belong together and often are not supported together (atomics/c11_atomics and work_groups/non_uniform_work_groups). Could we reconsider the need for this change?

**Comment by bashbaug** [comment]
Related issue: https://github.com/KhronosGroup/OpenCL-CTS/issues/683

After working group review we decided to only merge the device timers test into the API test, for now.

Closing this PR.

**Pull request closed without merging**